Sources/KeyboardSupport/KeyboardAccessory.swift

```diff
@@ -29,5 +29,17 @@ public typealias KeyboardInputAccessory = KeyboardAccessory
 
 /// Represents something that has keyboard accessory navigation options.
```

Comment:
Now that we've broken this protocol apart, we should update the comments.

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/f0fb818e19adc3fe77eecb915f01f9ef479cc927